src/pages/New.js

```diff
@@ -26,14 +26,23 @@ export default function New() {
       imgSrc:
         "https://static.lieferando.de/images/restaurants/de/OPROO0P/logo_465x320.png",
       title: title,
-      categories: ["pizza", "salad", "pasta"],
+      categories: cuisine,
       distance: 6,
       rating: 3.1,
       description: "Eat italian"
     });
   }
 
   const [title, setTitle] = useState("");
+  const [cuisine, setCuisines] = useState([]);
+  function handleClickCuisine(newCuisine) {
+    const cuisines = [...cuisine];
+    cuisines.includes(newCuisine)
```

Comment:
jconditional operators are very handy. but try to use them only in declarations for better readability:

OK:
```js
const someObject = isConditionTrue ? oneObject : anotherObject;
```

Uncommon:
```js
isConditionTrue ? doSomething() : doSomethingElse();
```

I would prefer to have a simple `if` `else`  here.

```js
if (cuisines.includes(newCuisine)) {
  cuisines.splice(cuisines.indexOf(newCuisine), 1);
} else {
  cuisines.push(newCuisine);
}
```

See https://stackoverflow.com/a/6260001